packages/@lwc/template-compiler/src/__tests__/fixtures/svg/xlink-with-literal-value/expected.js

```diff
@@ -1,33 +1,8 @@
-import { registerTemplate, sanitizeAttribute } from "lwc";
-const stc0 = {
-  classMap: {
-    "slds-icon": true,
-  },
-  attrs: {
-    "aria-hidden": "true",
-    title: "when needed",
-  },
-  key: 0,
-  svg: true,
-};
+import { parseFragment, registerTemplate } from "lwc";
+const $fragment1 = parseFragment`<svg aria-hidden="true" class="slds-icon${0}" title="when needed"${2}><use xlink:href="/assets/icons/standard-sprite/svg/symbols.svg#case"${1}${2}></use></svg>`;
```

Comment:
The `xlink:href` stops going through the `sanitizeAttribute` hook, which is quite problematic. I don't think that this node should be static.